internal/explorer.point/src/pages/Wallet.jsx

```diff
@@ -124,38 +165,112 @@ export default function Wallet() {
         );
     };
 
+    const renderToken = (token, network) => {
```

Comment:
I don't understand why we keep avoiding react app best practices so hard. What is the reason for this function to be declared in the body of another component instead of being a separate functional one? What are the advantages such an approach provides?